src/multi-select/tp-multi-select.ts

```diff
@@ -107,8 +107,13 @@ export class TPMultiSelectElement extends HTMLElement {
 		const value: string[] = [];
 
 		const selectedOptions: NodeListOf<HTMLOptionElement> | null = this.querySelectorAll( 'select option[selected]' );
-		selectedOptions?.forEach( ( option: HTMLOptionElement ) => value.push( option.value ) );
-
+		selectedOptions?.forEach( ( option: HTMLOptionElement ) => {
+			const isSelected = 'selected' === option.getAttribute( 'selected' );
```

Comment:
@i-am-chitti Could you please check if this is required?

Reply:
It's not required. Have removed it.